ianalyzer_readers/extract.py

```diff
@@ -586,3 +591,72 @@ def _apply(self, metadata, *nargs, **kwargs):
         Extract `associated_file` from metadata and call `self.stream_handler` with file stream.
         '''
         return self.stream_handler(open(metadata['associated_file'], 'r'))
+
+
+class RDF(Extractor):
+    ''' An extractor to extract data from RDF triples
+    Parameters:
+        predicates: 
+            an iteratble of predicates with which to query for triples of interest
+        node_type:
+            either the subject or object of the triple is returned
+            default: 'object'; returning the subject instead can be useful, e.g., for identifiers
+        multiple: 
+            if `True`, return a list of all nodes for which the query returns a result,
+            if `False`, return the first node matching a query
+        is_collection:
+            specify whether the data of interest is a collection, i.e., sequential data
+            this is specified by the predicates `rdf:first` and `rdf:rest`, see https://rdflib.readthedocs.io/en/stable/_modules/rdflib/collection.html
```

Comment:
This docstring (and several others) is not rendered properly in the documentation because it's missing whitespace. There should be a whiteline before the "Parameters:" block.

This package uses [google-style docstrings](https://google.github.io/styleguide/pyguide.html#383-functions-and-methods); MKDocs is configured for this format. I'm not sure if this is documented properly in the contribution guidelines, I'll check that later.